motioneye/static/js/motioneye.de.json

```
@@ -117,7 +117,9 @@
 ,"Forigi ĉiujn": "Entfernen Sie sie alle"
 ,"Bildoj prenitaj de ": "Bilder von"
 ,"Filmoj registritaj de ": "Filme aufgenommen von"
-,"plena ekrano kamerao": "Vollbildkamera"
+,"montru ĉi tiun fotilon plenekranan": "Zeigen Sie diese Kamera Vollzeit"
```

```suggestion
,"montru ĉi tiun fotilon plenekranan": "Diese Kamera im Vollbildmodus anzeigen"
```
Changed in sources but not in JSON yet. A re`make` would do as well, but here the quick manual update 🙂.